ws-nextjs-app/pages/[service]/new_live/[id]/Post/index.tsx

```diff
@@ -1,66 +1,56 @@
-import React from 'react';
-
+/** @jsx jsx */
+import { jsx } from '@emotion/react';
 import pathOr from 'ramda/src/pathOr';
 import { OptimoBlock } from '#models/types/optimo';
-import headings from '#app/legacy/containers/Headings';
+import Heading from '#app/components/Heading';
 import Blocks from '#app/legacy/containers/Blocks';
-import paragraph from '#app/legacy/containers/Paragraph';
-import Text from '#app/components/Text';
-import { Post as PostType } from './types';
-
-// temporary solution to render LI/ OL blocks.
-const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
-  const listItems: (string | null)[] = blocks
-    .map(item =>
-      pathOr<string | null>(
-        null,
-        ['model', 'blocks', 0, 'model', 'text'],
-        item,
-      ),
-    )
-    .filter(text => typeof text === 'string');
-
-  if (listItems.length === 0) return null;
-
-  return (
-    <Text>
-      <ul>
-        {listItems.map((item, index) => (
-          // eslint-disable-next-line react/no-array-index-key
-          <li key={index}>{item}</li>
-        ))}
-      </ul>
-    </Text>
-  );
-};
+import Paragraph from '#app/legacy/containers/Paragraph';
```

Comment:
I am not sure if you have already look into this, but is it possible to use the new Text component instead of the paragraph here?

Reply:
So the issue with the new text (and paragraph) components is that (as far as I can tell) they don't handle nested fragments.

E.g. this is the data we might get in that we need to render:

```
{
                "type": "paragraph",
                "model": {
                  "text": "Another short form text",
                  "blocks": [
                    {
                      "type": "fragment",
                      "model": { "text": "Another ", "attributes": ["bold"] }
                    },
                    {
                      "type": "fragment",
                      "model": {
                        "text": "short form text",
                        "attributes": ["italic"]
                      }
                    }
                  ]
                }
              },
```

The 'blocks' logic in the legacy Paragraph container can handle these fragments, which allow bold, italic and inline links. I believe the plan is to eventually use our new components to handle this (as mentioned in a comment above).